chatbot.py

```diff
@@ -13,8 +14,7 @@
 
 @cl.on_chat_start
 def setup_multiple_chains():
-    llm = OpenAI(model='gpt-3.5-turbo-instruct',
-                 temperature=0)
+    llm = ChatOpenAI( model="gpt-4o")
```

Comment:
I have a few comments regarding this. -> 

- instead of specifying get-4o here, please update to:
llm = ChatOpenAI(model='gpt-3.5-turbo-16k')
This can very well work with gpt-4o, but when I tested it out, results were not as good as with the 3.5-turbo. 
- You can include a hash comment above saying this. can also work with gpu-4o
- Remove space between '(' and 'model' in compliance with PEP8 standards.